wlc/test_wlc.py

```diff
@@ -135,18 +137,111 @@ def test_repr(self):
+        resp = Weblate().create_project(
+            "Hello", "hello", "http://example.com/", "Malayalam", "ml"
+        )
+        self.assertEqual("Hello", resp["name"])
+        self.assertEqual("hello", resp["slug"])
+        self.assertEqual("http://example.com/", resp["web"])
+        self.assertEqual("Malayalam", resp["source_language"]["name"])
+        self.assertEqual("ml", resp["source_language"]["code"])
+
+    def test_create_language(self):
+        resp = Weblate().create_language(
+            name="Test Language",
+            code="tst",
+            direction="rtl",
+            plural={"number": 2, "formula": "n != 1"},
+        )
+        self.assertEqual("Test Language", resp["name"])
+        self.assertEqual("tst", resp["code"])
+        self.assertEqual("rtl", resp["direction"])
+        self.assertEqual(2, resp["plural"]["number"])
+        self.assertEqual("n != 1", resp["plural"]["formula"])
+
+    def test_create_component(self):
+        resp = Weblate().create_component(
+            project="hello",
+            branch="master",
+            file_format="po",
+            filemask="po/*.po",
+            git_export="",
+            license="",
+            license_url="",
+            name="Weblate",
+            slug="weblate",
+            repo="file:///home/nijel/work/weblate-hello",
+            template="",
+            new_base="",
+            vcs="git",
+        )
+        self.assertEqual("Hello", resp["project"]["name"])
+        self.assertEqual("hello", resp["project"]["slug"])
+        self.assertEqual("Weblate", resp["name"])
+        self.assertEqual("weblate", resp["slug"])
+        self.assertEqual("file:///home/nijel/work/weblate-hello", resp["repo"])
+        self.assertEqual("http://example.com/git/hello/weblate/", resp["git_export"])
+        self.assertEqual("master", resp["branch"])
+        self.assertEqual("po/*.po", resp["filemask"])
+        self.assertEqual("git", resp["vcs"])
+        self.assertEqual("po", resp["file_format"])
+
+        with self.assertRaisesRegex(WeblateException, "required"):
+            Weblate().create_component(project="hello")
+
+        with self.assertRaisesRegex(WeblateException, "required"):
+            Weblate().create_component(project="hello", name="Weblate")
+
+        with self.assertRaisesRegex(WeblateException, "required"):
+            Weblate().create_component(project="hello", name="Weblate", slug="weblate")
+
+        with self.assertRaisesRegex(WeblateException, "required"):
+            Weblate().create_component(
+                project="hello", name="Weblate", slug="weblate", file_format="po"
+            )
+
+        with self.assertRaisesRegex(WeblateException, "required"):
+            Weblate().create_component(
+                project="hello",
+                name="Weblate",
+                slug="weblate",
+                file_format="po",
+                filemask="po/*.po",
+            )
+
+
+class ObjectTestBaseClass:
```

Comment:
This does not inherit from TestCase causing some pytlint warnings: https://github.com/WeblateOrg/wlc/pull/82/checks?check_run_id=1872660001